karols-frontend/src/actions/actions.js

```diff
@@ -0,0 +1,29 @@
+export const SELECTED_CHOICESHOP = "CHOSE_CHOICESHOP"
```

Comment:
Be consistent with your actions type name 
- They must be the same as const name ``` const ANY_ACTION = 'ANY_ACTION'
- They must follow the same logic, you have to choose between SELECTED / CHOICE / CHOOSE